### ISSUE TITLE
[Question] Setting radioToUse

### ISSUE THREAD

**Issue by jpdias** [rfquack/RFQuack-cli#5]
Hello. I'm trying to use the out-of-the-box modules provided by RFQuack, but I can't manage to change the default radio to another one. 

As an example, I have connected two radios (radioA, which is a CC1101 and radioB, which is an NRF24). As I want to use the radioB (1), and the default is the radioA (0), how can I change them? 

After going trough the documentation and code, I couldn't find a way of passing a value of the type "rfquack_WhichRadioValue" by using the CLI. I hope that the attached image helps understand the situation.

![image](https://user-images.githubusercontent.com/3944845/101381764-0f2d6e80-38af-11eb-887b-9979278bad8a.png)

Thanks!

**Comment by FiorixF1** [comment]
Hi,

This should work

```python
wr = pb.__dict__["WhichRadioValue"]()
wr.value = 1   # 0 for RadioA, 1 for RadioB ...
q.roll_jam.listen_radio = wr
```

**Comment by jpdias** [comment]
Hey @FiorixF1, 

It really did work! But shouldn't there be any kind of macros for setting the radios?

Thanks!

**Comment by phretor** [comment]
@jpdias yes, having a macro could be useful. Unfortunately we don't have much time to dedicate to this project, so we're hoping that someone with a big ❤️ will do it, eventually 😄

**Comment by phretor** [comment]
Leaving this open as a reminder to implement a shortcut function.
